sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/TableCatalog.java

```diff
@@ -78,6 +80,11 @@ public interface TableCatalog extends CatalogPlugin {
    */
   String OPTION_PREFIX = "option.";
 
+  /**
+   * @return the set of capabilities for this TableCatalog
+   */
+  default Set<TableCatalogCapability> capabilities() {return Collections.emptySet();}
```

Comment:
Hmm. I thought we have `Java Linter` rule for this. Shall we add more `space` characters?
```java
{return Collections.emptySet();}
{ return Collections.emptySet(); }
```